src/connections.jl

```diff
@@ -68,8 +68,9 @@ base_manifold(F::LeviCivitaConnection) = F.manifold
 @doc raw"""
     apply_operator(F::LeviCivitaConnection, p, X, Y)
 
-Compute the value of the Levi-Civita connection at point `p`, in the direction pointed by
-tangent vector `X` at `p`, of the vector field on `F.manifold` defined by a function `Y`.
+Compute the value of the Levi-Civita connection of type [`LeviCivitaConnection`](@ref) at
+point `p`, in the direction pointed by tangent vector `X` at `p`, of the vector field on
+`F.manifold` defined by a function `Y`.
 The formula reads $(\nabla_X \mathit{Y})_p$.
```

Comment:
Why the `mathit` here? We should also be careful for the Levi Cevita, since it usually applies to vector fields not to vectors, then your notation might be confusing here.

Reply:
Right, that `mathit` is not necessary. Connections look pretty well defined for `X` being just a tangent vector but `Y` really needs to be a vector field (and it is in the tests). Do you have a better notation for vector fields?

Reply:
We basically only have capital greek letters left, right? If we write upfront $X,Y \in \mathcal{X}(\mathcal M)$ it might help and we might use X,Y for both vectors and vector fields, but all this should also be added to the notations.md.

Reply:
Mostly, we already use `\Xi` for a set of tangent vectors. I have written a separate response below regarding notation.

Reply:
Ok, then lets use X and Y and phrase is carefully in the doc text.